ExampleMod/Items/Weapons/ExampleYoyo.cs

```diff
@@ -38,6 +39,25 @@ public class ExampleYoyo : ModItem
 			item.shoot = ProjectileType<ExampleYoyoProjectile>();
 		}
 
+		// Make sure that your item can even receive these prefixes (check the vanilla wiki on prefixes)
```

Comment:
s/even/never?

Reply:
Basically, why putting prefixes in that array, if this item can't even get that prefix assigned by the game anyway? Certain item types can get different prefixes (weapons vs accessories, knockback vs no knockback etc.)